### ISSUE TITLE
Refinement follow-ups

### ISSUE THREAD

**Issue by mcaceresb** [gphk-metrics/stata-multe#4]
- [x] Add vcov matrix if possible.
- [x] Add printing for decomposition results (check the format).
- [x] Add full reference to working paper.
- [x] Check any other references to the help file as needed.
- [X] Code post-estimation commands (e.g. computing lambda and tauhat on the fly post estimation by the user).
    - NB: Post-estimation commands were set aside for now.
- [x] Confirm the SE decomposition is hetero robust.
- [x] Clarify optional arguments in help file.
- [x] Notify the team once this is merged into `main`.
